index.ts

```diff
@@ -84,7 +84,7 @@ export function gretch<T = DefaultGretchResponse, A = DefaultGretchError>(
     ...(rest as RequestInit),
   };
   const controller =
-    typeof AbortController !== "undefined" ? new AbortController() : null;
+    typeof AbortController !== "undefined" ? new AbortController() : undefined;
```

Comment:
TS didn't like an optional param (evals to `Type | undefined`) being potentially `null`